controllers/authors.js

```diff
@@ -0,0 +1,30 @@
+const models = require('../models')
+
+const getAllAuthors = async (request, response) => {
+  try {
+    const authors = await models.authors.findAll()
+
+    return response.send(authors)
+  } catch (error) {
+    return response.status(500).send('Unable to retrieve authors, try again')
+  }
+}
+
+const getAuthorById = async (request, response) => {
+  try {
+    const { id } = request.params
+
+    const author = await models.authors.findOne({
+      where: { id },
+      include: [{ model: models.novels, include: { model: models.genres } }]
+    })
+
+    return author
+      ? response.send(author)
+      : response.sendStatus(404).send(`Could not find an author with a matching id of ${id}`)
+  } catch (error) {
+    return response.status(500).send('Unable to retrieve author, try again')
+  }
+}
```

Comment:
`response.sendStatus` sends an empty response with that status, it does not have a `.send` method

`response.status` sets the status on the response object and returns that new object which allows you to call `.send` on the returned response object